src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -17,10 +21,21 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
+	const { issue_key: key, issue_summary: summary } = req.query;
+	if (!key) {
+		return next(new Error(Errors.MISSING_ISSUE_KEY));
+	}
+
+	const branchSuffix = summary ? replaceSpaceWithHyphenHelper(replaceNonAlphaNumericWithHyphenHelper(summary as string)) : "";
```

Comment:
Should we put limit on branch name length?

Reply:
we had a google and github has no limit so its a free for all

Reply:
@joshkay10 , we'll end up with a very long branch name in some cases. I think, it is better to truncate description to have some relevant branch name. your thoughts?